hw06-atm/src/main/java/ru/otus/homework/ATM.java

```diff
@@ -0,0 +1,145 @@
+package ru.otus.homework;
+
+import lombok.extern.java.Log;
+import org.apache.commons.collections4.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.EnumSet;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+/**
+ * Class for ATM, which holds a number of Cell(s) with default values or custom ones
+ */
+@Log
+public class ATM {
+    private final Comparator<Cell> compareCells = Comparator.comparing(Cell::getNominal, Comparator.reverseOrder());
+    private static final EnumSet<DEFAULT_NOMINALS> defaultNominals = EnumSet.allOf(DEFAULT_NOMINALS.class);
```

Comment:
```suggestion
    private static final Set<Nominal> DEFAULT_NOMINALS = EnumSet.allOf(DEFAULT_NOMINALS.class);
```

Nominal - это ваш, переименованный DEFAULT_NOMINALS